internal/support/firefox_log_parser.py

```diff
@@ -446,14 +465,15 @@ def socket_transport_entry(self, msg):
                 port = match.groupdict().get('port')
                 self.http['sockets'][socket] = {'host': host, 'port': port}
         # nsSocketTransport::SendStatus [this=143f4000 status=804b0007]
+        # nsSocketTransport::SendStatus [this=7fe074bd2a00 status=4B0007]
         elif msg['message'].startswith('nsSocketTransport::SendStatus '):
             match = re.search(r'^nsSocketTransport::SendStatus \['
                               r'this=(?P<socket>[\w\d]+) '
                               r'status=(?P<status>[\w\d]+)', msg['message'])
             if match:
                 socket = match.groupdict().get('socket')
                 status = match.groupdict().get('status')
-                if status == '804b0007':
+                if status in ['804b0007', '4b0007']:
```

Comment:
Added this based on [nsresult::NS_NET_STATUS_CONNECTING_TO](https://searchfox.org/mozilla-central/source/__GENERATED__/xpcom/base/ErrorList.h#169)